sqlserver/metadata.csv

```diff
@@ -39,6 +39,16 @@ sqlserver.database.avg_fragmentation_in_percent,gauge,,,,"Logical fragmentation
 sqlserver.database.backup_count,gauge,,,,"The total count of successful backups made for a database.",0,sql_server,backup count
 sqlserver.database.fragment_count,gauge,,,,"The number of fragments in the leaf level of an IN_ROW_DATA allocation unit.",0,sql_server,fragment count
 sqlserver.database.is_sync_with_backup,gauge,,,,"Whether or not the database is marked for replication synchronization with backup. 0 = Not marked for replication sync, 1 = Marked for replication sync.",0,sql_server,is sync with backup
+sqlserver.files.reads,count,,read,second,"Number of reads issued on the file. Tags: `logical_name`, `file_location`, `database`, `state`",0,sql_server,reads on database file
```

Comment:
if we plan on making these metrics only for DBM customers we should add `(DBM Only)` to these descriptions